src/model/generated/backstopPool.model.ts

```diff
@@ -21,13 +24,13 @@ export class BackstopPool {
     @PrimaryColumn_()
     id!: string
 
-    @Column_('text', { nullable: false })
```

Comment:
I assume all of these changes in the model are from a prior dependency change (not included in this particular PR) but we never ran the model generation again (?) Looks like an improvement to me though.

Reply:
Yes most likely because of an update and running the generation from the schema again. I only added those 3 instances I referenced in the description.